pygfx/materials/_base.py

```diff
@@ -567,6 +567,9 @@ def render_queue(self) -> int:
 
         Objects with ``render_queue`` between 1501 and 2500 are sorted front-to-back. Otherwise
         objects are sorted back-to-front.
+
+        It can be tempting to use ``material.render_queue`` to control the render order of individual objects,
+        but for that purpose ``ob.render_order`` is more appropriate.
```

Comment:
I felt this in https://github.com/fastplotlib/fastplotlib/pull/873

Reply:
Render queues are solving a different problem :)